src/Chapter2.hs

```diff
@@ -780,13 +801,15 @@ Let's now try to eta-reduce some of the functions and ensure that we
 mastered the skill of eta-reducing.
 -}
 divideTenBy :: Int -> Int
-divideTenBy x = div 10 x
+divideTenBy = div 10
 
 -- TODO: type ;)
-listElementsLessThan x l = filter (< x) l
+listElementsLessThan :: Int -> [Int] -> [Int]
+listElementsLessThan x = filter (< x)
 
 -- Can you eta-reduce this one???
-pairMul xs ys = zipWith (*) xs ys
+pairMul :: [Int] -> [Int] -> [Int] 
+pairMul xs = zipWith (*) xs
```

Comment:
Great! 🏆 
After `ys` is eta-reduced, you can do the same with `xs` as well. So it could be even simpler: 
```suggestion
pairMul = zipWith (*)
```